Jenkinsfile

```diff
@@ -98,6 +115,26 @@ pipeline {
         } }
       }
     }
+    stage('Smoke Test') {
+      when { allOf {
+        expression { return !PIPELINE_CONTROL.ci_skip }
+      } }
+      steps {
+        timeout(time: 10, unit: 'MINUTES') { script {
+          def vscodePackageJson = readJSON file: "package.json"
+          def version = "v${vscodePackageJson.version}"
+          sh "git tag ${version}"
+
+          sh "npx vsce package -o ${version}.vsix"
+
+          // Release to Artifactory
+          withCredentials([usernamePassword(credentialsId: ARTIFACTORY_CREDENTIALS_ID, usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) { script {
+            def uploadUrlArtifactory = "https://zowe.jfrog.io/zowe/libs-snapshot-local/org/zowe/vscode/${version}.vsix"
+            sh "curl -u ${USERNAME}:${PASSWORD} --data-binary -H \"Content-Type: application/octet-stream\" -X PUT ${uploadUrlArtifactory} -T @${version}.vsix"
+          } }
```

Comment:
If the purpose of this stage is to "Archive Build artifacts in Zowe artifactory", do we need a separate stage, or could we move this logic to the build stage.
I'm on the fence on this one since it doesn't really affect too much to have lots of stages :)

Reply:
@jackjia-ibm, It'd be nice if we could overload the `archiveArtifacts` method to upload files to the zowe.jfrog.io site, (maybe on a default artifacts location, or the user-specified location in artifactory)

Reply:
@zFernand0 good idea. I created an issue https://github.com/zowe/jenkins-library/issues/57 to track it.